runtime/src/qio/qio_formatted.c

```diff
@@ -1107,13 +1110,17 @@ qioerr qio_channel_scan_literal(const int threadsafe, qio_channel_t* restrict ch
     if( err ) return err;
   }
 
-  err = qio_channel_mark(false, ch);
+  err = qio_channel_mark(false, ch); //A
```

Comment:
What are these 'A' and 'B' comments?

Reply:
Just to keep track of which pairs of mark+revert/commit go together. I can remove them if it seems more confusing than helpful.

Reply:
For me, the letters alone aren't immediately clear on an initial read-through, but I did assume it was some kind of pairing after reading some more. Maybe "begin mark A"? Anyways, I think it's fine either way.